checkbox-ng/docs/side-loading.rst

```diff
@@ -48,26 +47,28 @@ Create ``checkbox-providers`` directory in ``/var/tmp/``::
     You may not have write permissions for ``/var/tmp/``. You may want to
     run mkdir with sudo and later ``chown`` that directory
 
-Clone plainbox-provider-checkbox to the side-loaded directory::
+Clone Checkbox repository and copy the base provider to the side-loaded
+directory::
 
-    cd /var/tmp/checkbox-providers
-    git clone --depth=1 http://git.launchpad.net/plainbox-provider-checkbox
+    cd $HOME
+    git clone --depth=1 https://github.com/canonical/checkbox.git
+    cp -r $HOME/checkbox/providers/base /var/tmp/checkbox-providers/
 
 .. tip::
     --depth=1 tells git not to download all the history of the repo
 
 When started, Checkbox should display following warning::
 
-    $ checkbox-snappy.checkbox-cli
+    $ checkbox.checkbox-cli
     WARNING:plainbox.session.assistant:Using side-loaded provider:
-    com.canonical.certification:plainbox-provider-checkbox
+    com.canonical.certification:checkbox-provider-base
```

Comment:
Just tried, the print seems a bit different:
```
ubuntu@ubuntu-HP-ProBook-440-14-inch-G9-Notebook-PC:/var/tmp$ checkbox-cli
Using sideloaded provider: checkbox-provider-base, version 2.1.0 from /var/tmp/checkbox-providers/base
```

Reply:
Ah, nice! I had just manually updated the content, but you're right, with more recent versions of Checkbox, you get a different message. Thanks!